src/chats/components/conversation/footer/MessageComposer.tsx

```diff
@@ -375,17 +376,22 @@ const MessageComposer: React.FC<ConversationMessageComposerProps> = ({ roomId })
 		]
```

Comment:
why you use getUserSettings instead of useUserSettings?

Reply:
I thought that since the language cannot change dynamically, a subscription is useless